.jshintrc

```diff
@@ -30,6 +30,7 @@
     "define": true,
     "require": true,
     "Chromath": false,
-    "setImmediate": true
+    "setImmediate": true,
+    "$": false
```

Comment:
if your still unsure about what I mean, please google requirejs or look at some directive file in grafana that is also using jquery , and remove the change to jshintrc